models/time_utils/time_utils_minutes.sql

```diff
@@ -0,0 +1,7 @@
+{{ config( alias='minutes', tags=['static'], materialized = 'table')}}
+
+select explode(sequence(
+         date_trunc('minute', now() - interval '20 year')
```

Comment:
should we make the range fixed? I can't think any reason why a 40 year window shifting a few years would have any negative consequences on queries analysing crypto data, but feels like a best practice to make it truly static.

wdyt?

Reply:
I was assuming we'd just run this once but I guess we periodically rerun tables so it makes sense to make this fixed. I can update with a fixed time frame.